### DISCUSSION TITLE
New features

### DISCUSSION THREAD

**Discussion by Yarroudh** [Yarroudh/Optim3D#6] [General]
I'm working on implementing new features:

- The possibility to choose between OcTree and KDTree indexing for 3D point clouds. The performances of both structures will be compared.
- Integrate ZRect3D as a part of Optim3D post-processing. The buildings ground-floor elevation should be automatically corrected at the end of the process.
- The docker image is now available. Some issues were caught while reconstructing the buildings.
- The processing areas should consider the buildings that are in the limits between two tiles.

**Comment by Yarroudh** [comment]
@Mboga take a look at this. All of that would be implemented in the upcoming version.

**Reply by Mboga** [reply]
@Yarroudh this will be interesting !
